### DISCUSSION TITLE
CName Issues

### DISCUSSION THREAD

**Discussion by MatthewStridiron** [community/community#31669] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi.

I've been trying to connect a custom domain name to my personal website. Ever since then, my site has not been working. I realized after going through the logs that my website record is an A record, not a CNAME record. I'm just wondering how to convert this A Record to a CNAME record.  

![pic1](https://user-images.githubusercontent.com/25061903/188028225-a0bfb63e-7e00-4900-b2f7-64bc55f05b73.jpg)

I think based on previous posts (https://github.com/modcluster/docs.modcluster.io/issues/7), it should look like this. 
![pic2](https://user-images.githubusercontent.com/25061903/188028305-f2afe9c0-d8d0-4fba-bd7a-2cc539ae7f7c.jpg)

**Comment by toufiqul-hasan** [comment, marked as answer]
In your domain control panel:

go to Advance DNS then add these line:

A Record | @ | 185.199.108.153
A Record | @ | 185.199.109.153 
A Record | @ | 185.199.110.153 
A Record | @ | 185.199.111.153
CNAME Record | www | MatthewStridiron.github.io. 

save it.
